doc/api/n-api.md

```diff
@@ -2492,6 +2492,35 @@ This API creates a JavaScript `symbol` value from a UTF8-encoded C string.
 The JavaScript `symbol` type is described in [Section 19.4][]
 of the ECMAScript Language Specification.
 
+#### `node_api_symbol_for`
```

Comment:
Can we maybe alias all functions as `node_api_*` for consistency? We probably cannot rename the symbols since we don't want to break binary compatibility, but mixing the old and new names here is surprising. Maybe we can simply `#define` new names for all old functions and then use the new names in the documentation?

Reply:
Sounds like a good idea to me. Will do this in a separate PR if there are no objections. :+1:

Reply:
I think we had discussed doing that but if I remember correctly that could like to add-ons not compiling with earlier versions of Node.js. Either way +1 to a different PR just wanted to mention that we might not be able to do it as easily as we'd like.